src/component/mxgraph/style/utils.ts

```diff
@@ -110,12 +110,40 @@ export const updateFont = (cellStyle: string, font: Font): string => {
   return cellStyle;
 };
 
+const convertDirection = (direction: GradientDirection): string => {
+  switch (direction) {
+    case 'left-to-right':
+      return mxConstants.DIRECTION_EAST;
+    case 'right-to-left':
+      return mxConstants.DIRECTION_WEST;
+    case 'bottom-to-top':
+      return mxConstants.DIRECTION_NORTH;
+    case 'top-to-bottom':
+      return mxConstants.DIRECTION_SOUTH;
+    default:
+      return mxConstants.DIRECTION_EAST;
```

Comment:
**nitpick:** 
@tbouffard Now, the test coverage is [94.1% Coverage](https://sonarcloud.io/component_measures?id=process-analytics_bpmn-visualization-js&pullRequest=2760&metric=new_coverage&view=list), because we can't test it as the direction is mandatory 🙁

Reply:
You can remove the line related to the 'left-to-right', it will fall into the default case.

Alternatively, I am pretty sure that we can force the usage of undefined or any other string in tests by casting the passed value into `GradientDirection` or ignoring the TS error. This may not be needed with the proposal above.